events/room-booking.md

```diff
@@ -2,7 +2,7 @@
 Steps for room and AV (Audio Visual) cell booking for conducting workshops and other events.
 
 ## Steps for Room Booking
-- Go to the Junior Executive, Academic Section (UG) (Mr. Shouvik Adhikari as of now) and get the form for room booking.
+- Go to the Junior Executive, Academic Section (UG) (Mr. Shouvik Adhikari as of now) and get the form for room booking. [Here](../assets/room-booking-form.pdf) is a scan of the form.
 - This form is only for room booking and not for AV support.
 - Fill in the details necessary for booking a room. Date, time, and duration.
 - Get the signature of a Professor and the HOD of the respective department as the booking is made in the professor's name (Approach professors at the earliest). CSE professors are easily approachable for this, so can reach them out.
```

Comment:
- Mention about JC sir :)
- The booking must be finalised before 7 days of the event. Basically start the booking process 2 weeks earlier since there are many more things to do.
- Mention about the minimum time it will take to get a room booked 4-5 days.

Reply:
> Mention about JC sir :)

Is mentioning specific names fine (without consent that too)? 

> The booking must be finalised before 7 days of the event. Basically start the booking process 2 weeks earlier since there are many more things to do.

This point is mentioned below but i'll mention it here too.

> Mention about the minimum time it will take to get a room booked 4-5 days.

ok

Reply:
> Is mentioning specific names fine (without consent that too)?

what's wrong in it? he supports us and open source publicly, it's already evident ¯\_(ツ)_/¯

Reply:
Slack pe continue karte hain discussion.